### DISCUSSION TITLE
New Programes Data Ingestion - 400 Bad Request

### DISCUSSION THREAD

**Discussion by gprao08** [Sunbird-cQube/community#864] [General]
@RNarmatha 
@bommala
After run "VSK_Grammer" (totalteachers, teacherspresent, teachersmarked) I try to run dimension_schema I am getting '400 bad request'. Yesterday we have already upload the dimensions found in miniobucket process_input folder those are 1. district-dimension, 2. grade-dimension, 3. medium-dimension, 4. state-dimension, 5. subject-dimension. Now can I move without these 5 dimensions?

**Comment by bommalla** [comment]
@gprao08 

Thanks for reaching us !!

**Comment by bommalla** [comment]
@gprao08 

For new program ingestion we need below dimension.please keep 5 dimension  and upload remaining dimensions.
academicyear-dimension.data.csv
block-dimension.data.csv
cluster-dimension.data.csv
district-dimension.data.csv
exam-dimension.data.csv
grade-dimension.data.csv
medium-dimension.data.csv
schoolcategory-dimension.data.csv
school-dimension.data.csv
schoolmanagement-dimension.data.csv
statecategory-dimension.data.csv
state-dimension.data.csv
subject-dimension.data.csv

**Comment by gprao08** [comment]
@bommalla 
Hi Rajesh, school-dimension.data.csv not upload because of file size large. Is it correct?
[school-dimension.data.csv](https://github.com/Sunbird-cQube/community/files/14355573/school-dimension.data.csv)

**Reply by bommalla** [reply]
@gprao08 

It's uploaded but it will take some time.

**Reply by gprao08** [reply]
@bommalla 
From postman, it returns the error code 400 'bad request'. Even I try to upload manual from minio one file which I mentioned above not upload due to large size...

**Reply by bommalla** [reply]
@gprao08 
Please avoid manual uploads. Use Postman for uploading, please check you internet speed and ingest again.

**Comment by bommalla** [comment]
@gprao08 

I just checked your school-dimension file and noticed that two columns are missing(schoolmanagement_id,school_statecategory_id). Please add these two columns and proceed with the ingestion process again.

**Reply by gprao08** [reply]
@bommalla 
Hi Rajesh, found some discrepancy in other dimension files as well. Could you plz assign a call in this regard? So that I can clarify in detail?

**Reply by gprao08** [reply]
@bommalla 
Hi Rajesh, as discussed after run ingest_data (schedule api) identified the remaining dashboards also not visible. Could you plz look into it.

**Comment by gprao08** [comment]
@bommalla 
Thanks for your support, Now Teachers attendance dashboard and other dashboards showing counts.

**Reply by bommalla** [reply]
@gprao08 

Thanks for the update
